assets/styles/custom.scss

```diff
@@ -21,4 +22,5 @@
   --gray: #d4d4d4 !important;
   --lightgray: #292633 !important;
   --outlinegray: #343434 !important;
+  --million-progress-bar-color: var(--secondary) !important;
```

Comment:
i think this is redundant as this isn't being overwritten in dark mode

Reply:
i see... just a quick question on the design, do you want it to be kept as the original blue-ish color or the secondary color? I'll just hard code the hex values

Reply:
secondary colour is fine, i just mean L25 can be omitted as that set of styles only takes over in dark mode and `var(--secondary)` already changes when it toggles to dark

Reply:
gotcha 👍